bin/run_demo.sh

```diff
@@ -52,8 +52,8 @@ fi
 # can not be the same.
 # Do not specify a number of processors (-j) to test that the default value
 # works.
-pipetask run -d "exposure=903342 AND detector=10" -b DATA_REPO/butler.yaml \
-    -i HSC/calib,HSC/raw/all,refcats --longlog \
+pipetask --long-log run -d "exposure=903342 AND detector=10" -b DATA_REPO/butler.yaml \
```

Comment:
hmm, I thought we are keeping all options to be per-sub-command options?

Reply:
logging was an exception to this.

Reply:
I'm fine with that as long as there is a consensus. Initially `pipetask` had global options too, I'm glad we are going in circles 🙂